test_conformance/extensions/cl_khr_command_buffer/negative_command_buffer_svm_mem.cpp

```diff
@@ -50,6 +50,12 @@ struct CommandBufferCommandSVMQueueNotNull : public BasicSVMCommandBufferTest
     }
 
     const cl_char pattern_1 = 0x14;
+
+    bool Skip() override
+    {
+        return is_extension_available(device,
+                                      "cl_khr_command_buffer_multi_device");
```

Comment:
Think we should be doing an OR with the base class `Skip()` methods for these new additions, e.g. `BasicSVMCommandBufferTest::Skip()` here. I'm seeing this test not skip anymore on a device without SVM support.

Reply:
Thanks, should be fixed now